packages/hooks/src/useIsPasskeySupported.ts

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { platformAuthenticatorIsAvailable } from '@simplewebauthn/browser';
+import useGrowthbookFeatureFlag from './useGrowthbookFeatureFlag';
+
+const useIsPasskeySupported = () => {
+    const [is_passkey_supported, setIsPasskeySupported] = React.useState(false);
+    const [is_passkey_support_checking, setIsPasskeySupportChecking] = React.useState(true);
+    const is_passkeys_enabled = useGrowthbookFeatureFlag({
+        featureFlag: 'web_passkeys',
+        defaultValue: 'uninitialized',
+    });
+
+    React.useEffect(() => {
+        const checkPasskeySupport = async () => {
+            try {
+                const result = await platformAuthenticatorIsAvailable();
+                setIsPasskeySupported(result && !!is_passkeys_enabled);
+            } catch (error) {
+                /* eslint-disable no-console */
+                console.error('Error checking passkey support:', error);
```

Comment:
```suggestion
                if (!isProduction()) {
                        console.error('Error checking passkey support:', error);  // eslint-disable-line
                }
```
we don't want to get this in TrackJS

Reply:
cool, thanks 👍 , missed it